### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Import ExtensionModify Columns_Keyboard: The focus after using escape key on the list of items in column 2 row 2 drop down button is seen on the edit field beside it.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4175]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS;-A11yTCS;-A11ySev2;-BM_SQLAzureDataStudio_Oct2020; -SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;- WCAG2.4.3;-Benchmark;-Keyboard;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW use the right-click key on the keyboard on it and select import wizard.
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Tab to drop down button in cell column 2 row2 and press enter key.
7. Once the items have been expanded use the escape key to close the list.

**Actual:**
On using the escape key to close the list for the drop-down button in column 2 row2 the focus in on the edit field in column 2 row2 .

**Expected:**
The focus on using escape key on the list should be on the drop down button that opened the list.

**User Impact:**
The keyboard users will have to use an extra tab to move to the next column and this would disrupt the focus order.

**Attachment:**
